portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase

```diff
@@ -24,6 +24,8 @@ definition {
 	tearDown {
 		var testPortalInstance = PropsUtil.get("test.portal.instance");
 
+		ACUtils.tearDownDXP();
+
```

Comment:
Hey @manuferreira, could you add the `tearDown` macro from AC here following how it was used in this [other file](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsBlogs.testcase#L28) (same comment and commented macro in the code)?

Reply:
Done :heavy_check_mark: